near-sdk/src/promise.rs

```diff
@@ -751,11 +735,11 @@ impl Promise {
     /// and finally `dave_near` and `eva_near` are created concurrently.
     ///
     /// ```no_run
-    /// # use near_sdk::Promise;
-    /// let p1 = Promise::new("bob_near".parse().unwrap()).create_account();
-    /// let p2 = Promise::new("carol_near".parse().unwrap()).create_account();
-    /// let p3 = Promise::new("dave_near".parse().unwrap()).create_account();
-    /// let p4 = Promise::new("eva_near".parse().unwrap()).create_account();
+    /// # use near_sdk::{AccountId, Promise};
+    /// let p1 = Promise::new("bob_near".parse::<AccountId>().unwrap()).create_account();
```

Comment:
On one hand, I love explicitness, on another, it becomes a bit vebose and technically a breaking change that many existing contracts will hit. Can we land this PR for 6.x release or do you depend on something here that you need to have in 5.x?

With this explicit `AccountId` import, maybe it will be easier to read if we use `AccountIdRef::new_or_panic()` instead of `"".parse::<AccountId>().unwrap()`